src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import styles from './header.module.css';
+import NavBar from './NavBar'
+
+function Header() {
+
+    const navLinks = {
+        "/" : "Home",
+        "/projects" : "Projects",
+        "/contact": "Contact"
+    }
+
+  return (
+    <div className={styles.header}>
+        <h4><a href='/'>John E Mook</a></h4>
+        <NavBar 
```

Comment:
nit: You can't see it in the diff, but it looks like there's an extra space after `<NavBar` here:

```suggestion
        <NavBar
```